web/ui/dashboard/src/app/private/pages/project/events/event/event.component.html

```diff
@@ -145,3 +147,10 @@ <h3 class="text-16 font-bold mb-16px mt-26px" *ngIf="sidebarEventDeliveries && s
 		</svg>
 	</convoy-button>
 </div>
+
+
+<convoy-modal position="full" title="event payload" *ngIf="showPayload" (closeModal)="showPayload = false">
+    <div class="py-40px" modalBody>
+        <prism language="json" [code]="getCodeSnippetString()"></prism>
+    </div>
+</convoy-modal>
```

Comment:
So we repeat it everywhere?
Naa, it has to be treated from the prism component itself.